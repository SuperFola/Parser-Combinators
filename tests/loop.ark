#
(while 1 1)
(
while
2
2
)



(
    
      while  3  3  )




#