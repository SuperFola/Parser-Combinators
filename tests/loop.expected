( Keyword:while Number:1 Number:1 )
( Keyword:while Number:2 Number:2 )
( Keyword:while Number:3 Number:3 )